docs/source/manual/file_formats/reference_file.rst

```diff
@@ -0,0 +1,80 @@
+.. _file_format_reference_file:
+
+Reference File (.yaml)
+----------------------------------------
+
+This file is generated by command :ref:`openfpga_setup_commands_report_reference`
+
+
+The reference file aims to the show reference number of each child module of given parent module
+
+By using the options of the command :ref:`openfpga_setup_commands_report_reference`, user can selectively output the reference info under the given parent module on their needs.
+
+An example of the file is shown as follows.
+
+.. code-block:: yaml
+
+Date: Mon Sep  9 16:41:53 2024
+
+#the instance names are given during netlist generation
```

Comment:
Any code under the ``code-block`` should adapt indent (at least 2 spaces). Otherwise, you see the format issues of current build:
![image](https://github.com/user-attachments/assets/6af62c12-76d6-4aff-90eb-6b0ea0193e57)